notebooks/official/workbench/gaming_churn_prediction/churn_prediction_for_game_developers.ipynb

```diff
@@ -127,11 +101,11 @@
+        "This notebook uses [this public BigQuery dataset](https://console.cloud.google.com/bigquery?p=firebase-public-project&d=analytics_153293282&t=events_20181003&page=table), which contains raw event data from a real mobile gaming app called Flood It. ([Android app](https://play.google.com/store/apps/details?id=com.labpixies.flood), [iOS app](https://itunes.apple.com/us/app/flood-it!/id476943146?mt=8)). The [data schema](https://support.google.com/analytics/answer/7029846) originates from Google Analytics for Firebase, but is the same schema as [Google Analytics 4](https://support.google.com/analytics/answer/9358801); the techniques in this notebook can be applied to either Google Analytics for Firebase or Google Analytics 4 data.\n",
         "\n",
-        "Google Analytics 4 (GA4) uses an [event-based](https://support.google.com/analytics/answer/9322688) measurement model. Events provide insight on what is happening in an app or on a website, such as user actions, system events, or errors. Every row in the dataset is an event, with various characteristics relevant to that event stored in a nested format within the row. While Google Analytics logs many types of events already by default, developers can also customize the types of events they also wish to log.\n",
+        "Google Analytics 4 (GA4) uses an [event-based](https://support.google.com/analytics/answer/9322688) measurement model. Events offer insights into app or website activities, including user actions, system events, or errors. Each row in the dataset represents an event, with its various characteristics stored in a nested format within the row. Google Analytics logs many types of events by default, but developers can also customize and log additional event types as needed.\n",
         "\n",
-        "Note that as you cannot simply use the raw event data to train a machine learning model, this notebook shows you some important steps of how to pre-process the raw data into an appropriate format to use as training data for classification models."
+        "Note that you can't use raw event data directly to train a machine learning model. This notebook demonstrates some important steps to pre-process raw data into an appropriate format for training classification models."
```

Comment:
change: "Note that you can't use raw event data directly to train a machine learning model."
to: "Note: You can't use raw event data directly to train a machine learning model."